src/json.c

```diff
@@ -127,19 +124,15 @@ int Document_LoadSchemaFieldJson(Document *doc, RedisSearchCtx *sctx) {
 
     // on crdt the return value might be the underline value, we must copy it!!!
     // TODO: change `fs->text` to support hash or json not RedisModuleString
-    if (japi->getRedisModuleString(json, &doc->fields[oix].text) != REDISMODULE_OK) {
+    if (japi->getJSON(json, ctx, &doc->fields[oix].text) != REDISMODULE_OK) {
```

Comment:
`getJSON` or `getString`?

Reply:
`getRedisModuleString` was returning a JSON String. Now we renamed it to `getJSON`